resources/views/login.blade.php

```diff
@@ -0,0 +1,9 @@
+@extends('layouts/main-login')
+@section('content-body')
+    <div class="content">
+        <v-lnb></v-lnb>
```

Comment:
Can we please write readable variable names, even if its long, its better than short unreadable variables. Lets update line 4-7 in this file